src/views/states/list/components/StateRow.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { FC } from 'react';
+import { NodeNetworkStateModelGroupVersionKind } from 'src/console-models';
+
+import { ResourceLink, RowProps, TableData } from '@openshift-console/dynamic-plugin-sdk';
+import { List, ListItem } from '@patternfly/react-core';
+import { V1beta1NodeNetworkState } from '@types';
+
+const StateRow: FC<RowProps<V1beta1NodeNetworkState>> = ({ obj, activeColumnIDs }) => {
+  const typesCount = obj?.status?.currentState?.interfaces.reduce((acc, iface) => {
+    if (acc[iface.type]) {
+      acc[iface.type] += 1;
+    } else {
+      acc[iface.type] = 1;
+    }
+    return acc;
+  }, {} as { [key: string]: number });
+
+  return (
+    <>
+      <TableData id="name" activeColumnIDs={activeColumnIDs} className="pf-m-width-30">
+        <ResourceLink
+          groupVersionKind={NodeNetworkStateModelGroupVersionKind}
+          name={obj.metadata.name}
```

Comment:
I can see, in the line 9, you are using "?." to access various properties of `obj`. Is the situation different here? Is `obj` always defined? Is `metadata.name` always present? If yes, then please ignore this comment ;)

Reply:
Yes, the object is always defined and always has `metadata.name`. In line 9 we have all those `?` because I'm not sure if `currentState` prop is always defined.